packages/cli/src/lib/config/config.ts

```diff
@@ -48,10 +49,7 @@ export class Config {
     'region',
     'analyticsEnabled',
   ];
-  public constructor(
-    private configDir: string,
-    private error = console.error
-  ) {}
+  public constructor(private configDir: string) {}
```

Comment:
Reason for these change:
`console.error` does not exit the process as opposed to `cliUx.ux.error`. To prevent different behaviour based on the console log function, I attributed the logging work to CliUx as it is part of the `@oclif/core` package.